src/models/Offer.js

```diff
@@ -104,6 +110,11 @@ function validateJobMaxDuration(value) {
     return value >= this.jobMinDuration;
 }
 
+async function validateOwnerConcurrentOffers(value) {
+    const isOwnerValid = await concurrentOffersNotExceeded(this.constructor)(value, this.publishDate, this.publishEndDate);
+    return isOwnerValid;
+}
```

Comment:
```suggestion
function validateOwnerConcurrentOffers(value) {
    return concurrentOffersNotExceeded(this.constructor)(value, this.publishDate, this.publishEndDate)
}
```

This name is too generic. Luckily we don't need a variable here :P